css/page-styles/login.css

```diff
@@ -0,0 +1,71 @@
+@import url(../utilities/variables.css);
+@import url(../layouts/navbar.css);
+@import url(../layouts/footer.css);
+
+
+html{
+    background-color: var(--primary-transparent);
+}
+body{
+    font-family: var(--lato-font);
+}
+nav, footer{
+    font-family: var(--acme-font);
+}
+.container{
+    width: 100%;
+}
+.login-card{
+   display: flex;
+   flex-direction: column;
+   width: 28rem;
+   border-radius: 10px;
+   background-color: var(--white);
+   box-shadow: var(--box-shadow-1);
+   height: fit-content;
+   padding: 1rem 1.5rem ;
```

Comment:
You could declare variables for different common space units like 1 rem

Reply:
Updated the spacing size with its variables in new commit.